src/bp/core/services/stats/index.ts

```
@@ -0,0 +1,125 @@
+        botpressVersion: process.BOTPRESS_VERSION,
+        fingerprint: await this.getServerFingerprint(),
+        clusterEnabled: !(process.CLUSTER_ENABLED === null),
+        machineUUID: await machineUUID(),
+        nodesCount: await this.jobService.getNumberOfSubscribers()
+      },
+      license: {
+        type: this.getLicenseType(),
+        status: await this.getLicenseStatus()
+      }
+    }
+  }
+
+  private async getBotsCount(): Promise<number> {
+    const botIds = await this.botService.getBotsIds()
+    return botIds.length
+  }
+
+  private async getFlowCount(): Promise<number> {
+    const flows = await this.ghostService.bots().directoryListing('/', '*/flows/*.flow.json', '*/flows/error.flow.json')
+    return flows.length
+  }
+
+  private async getIntentsCount(): Promise<number> {
+    const intents = await this.ghostService.bots().directoryListing('/', '*/intents/*')
+    return intents.length
+  }
+
+  private async getContentElementsCount(): Promise<number> {
+    const contentElementFiles = await this.ghostService.bots().directoryListing('/', '*/content-elements/*')
+    const contentElements = await Promise.all(
+      contentElementFiles.map(filename => {
+        const obj = path.parse(filename)
+        return this.ghostService.bots().readFileAsObject<any[]>(obj.dir, obj.base)
+      })
+    )
+
+    return contentElements.reduce((acc, contentElementsArray) => {
+      return acc + contentElementsArray.length
+    }, 0)
+  }
+
+  private getLicenseType(): string {
+    return process.IS_PRO_ENABLED ? 'pro' : 'ce'
+  }
+
+  private async getServerFingerprint(): Promise<string | null> {
+    try {
+      return await this.licenseService.getFingerprint('cluster_url')
+    } catch (err) {
+      if (err.message && err.message === 'Not implemented') {
+        // tslint:disable-next-line: no-null-keyword
+        return null
+      }
+
+      throw err
+    }
```

I don't think the stats service should throw an error when fetching any kind of data. It should just return N/A